src/seed_tools/utils/studies_to_seed.ts

```diff
@@ -187,12 +187,8 @@ async function checkAndOptionallyFixFormat(
 }
 
 function setStudyDefaultParameters(study: Study) {
-  if (study.activation_type === undefined) {
```

Comment:
Was:
```
Running: eslint . --config src/.eslintrc.js

/Users/mikhail/work/brave-variations/src/seed_tools/utils/studies_to_seed.ts
  190:3  error  Prefer using nullish coalescing operator (`??=`) instead of an assignment expression, as it is simpler to read  @typescript-eslint/prefer-nullish-coalescing
  193:3  error  Prefer using nullish coalescing operator (`??=`) instead of an assignment expression, as it is simpler to read  @typescript-eslint/prefer-nullish-coalescing

✖ 2 problems (2 errors, 0 warnings)
```